src/index.ts

```diff
@@ -38,7 +38,20 @@ export {
   MONEY_MAKER_ADDRESS,
   VEJOE_STAKING_ADDRESS,
   VEJOE_TOKEN_ADDRESS,
-  BOOSTED_MASTERCHEF_ADDRESS
+  BOOSTED_MASTERCHEF_ADDRESS,
+  CURRENCY_MANAGER,
+  EXECUTION_MANAGER,
+  JOEPEG_EXCHANGE,
+  PROTOCOL_FEE_MANAGER,
+  ROYALTY_FEE_MANAGER,
+  ROYALTY_FEE_REGISTRY,
+  ROYALTY_FEE_SETTER,
+  STRATEGY_STANDARD_SALE_FOR_FIXED_PRICE,
+  TRANSFER_MANAGER_ERC721,
+  TRANSFER_MANAGER_ERC1155,
+  TRANSFER_MANAGER_NON_COMPLIANT_ERC721,
+  TRANSFER_SELECTOR_NFT,
+  ERC721_TOKEN
```

Comment:
can we append `_ADDRESS` to the end of these to be consistent w/ the other names?